File: analytics.js
var _gaq;

function InitGoogleAnalytics(ID){
	_gaq = _gaq || [];
	_gaq.push(['_setAccount', ID]);
 	_gaq.push(['_trackPageview']);
	_gaq.push(['_setDomainName', 'none']);
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
	
}

function GoogleAnalyticsEvent(Category,Action,Label,Value,NoTrack){

	if(!Label){
		_gaq.push(['_trackEvent', Category, Action]);
	}else if(!Value){
		_gaq.push(['_trackEvent', Category, Action, Label.m_value]);
	}else if(!NoTrack){
		_gaq.push(['_trackEvent', Category, Action, Label.m_value, Value.m_value]);
	}else{
		_gaq.push(['_trackEvent', Category, Action, Label.m_value, Value.m_value, NoTrack.m_value]);
	}

}

File: analytics.java
import com.google.analytics.tracking.android.GoogleAnalytics;
import com.google.analytics.tracking.android.Tracker;
import com.google.analytics.tracking.android.EasyTracker;

class Analytics{

	static Tracker GATracker;

	public static void InitGoogleAnalytics(String ID){
		EasyTracker.getInstance().activityStart(BBAndroidGame.AndroidGame().GetActivity());
		GATracker = GoogleAnalytics.getInstance(BBAndroidGame.AndroidGame().GetActivity()).getTracker(ID);
	}
	
	public static void GoogleAnalyticsEvent(String category,String action){
		GATracker.sendEvent(category, action, "", null);
	}
	
	public static void GoogleAnalyticsEvent(String category,String action,String label){
		GATracker.sendEvent(category, action, label, null);
	}	
	
	public static void GoogleAnalyticsEvent(String category,String action,String label,long number){
		GATracker.sendEvent(category, action, label,number);
	}
	
	public static void GoogleAnalyticsEvent(String category,String action,String label,long number,Boolean noTrack){
		GATracker.sendEvent(category, action, label,number);
	}		
				
}